examples/load_ui_base_instance.py

```diff
@@ -0,0 +1,40 @@
+import sys
+import os
+
+os.environ["QT_PREFERRED_BINDING"] = "PySide"
+from Qt import QtWidgets, load_ui
+
+
+def setup_ui(uifile, base_instance=None):
+    ui = load_ui(uifile)
+    if not base_instance:
+        return ui
+    else:
+        for member in dir(ui):
+            if not member.startswith('__') and \
+               member is not 'staticMetaObject':
+                setattr(base_instance, member, getattr(ui, member))
+        return ui
+
+
+class MainWindow(QtWidgets.QWidget):
+    def __init__(self, parent=None):
+        QtWidgets.QWidget.__init__(self, parent)
+        setup_ui('examples/load_ui_qwidget.ui', self)
+
+
+def test_load_ui_setup_ui_wrapper():
+    """Example: load_ui with setup_ui wrapper
+    """
+    app = QtWidgets.QApplication(sys.argv)
+    window = MainWindow()
+
+    # Tests
+    assert isinstance(window.__class__, type(QtWidgets.QWidget))
```

Comment:
True, I can change that into

    assert isinstance(window.__class__, QtWidgets.QWidget.__class__)